### PR TITLE
setiadi's blog post

### PR DESCRIPTION
**Pull request by diegoonolasco**

### CODE REVIEW

**Comment by mattwthompson** [comment]
I only skimmed but I noticed images 5 and 6 [aren't rendering for me](https://github.com/openforcefield/openforcefield.org/blob/90fa423f00d1609f76c97a47dcca55181b5873ce/content/community/news/science-updates/2022-08-29-setiadi-blog-post.md). Might be worth a double-check there - everything else looks fine to my eye

image

**Comment by diegoonolasco** [comment]
@mattwthompson, could you please check again?

**Comment by mattwthompson** [comment]
Looks great!

**Comment by diegoonolasco** [comment]
Thanks, buddy!

File: content/community/news/science-updates/2022-08-29-setiadi-blog-post.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+ 
+We see that reducing the oxygen GB radius significantly improves the binding free energy calculation. The root-mean-squared error (RMSE) decreases from 2.92 kcal/mol to 1.00 kcal/mol.
+
+<b>Final Remarks</b>
+
+In this blog, I have shown an example of optimizing GB radii to fit host-guest binding data. The results here show some promise, but a larger training set is needed for production. There are also several things to consider:
+Which parameters should be optimized (i.e. how sensitive are different GB parameters to binding affinity)?
+How transferable are the optimized FF parameters to other host-guest complexes?
+What happens to other physical properties when we only fit the FF parameters to HG binding?
+How well do host-guest fitted FF parameters perform in protein-ligand systems?
```
Comment (jeff231li) [inline content/community/news/science-updates/2022-08-29-setiadi-blog-post.md]: I think these lines are missing bullet points

Comment (diegoonolasco) [reply content/community/news/science-updates/2022-08-29-setiadi-blog-post.md]: Could you check again?

**Review by j-wags — APPROVED** [review]
I was having trouble getting the pictures to render as well, so I changed the folder name to be lower case so it lines up with the slug in the markdown. I also deleted the unused images. This looks good to me :-)

File: content/community/news/science-updates/2022-08-29-setiadi-blog-post.md (hunk 1 above)
Comment (jeff231li) [reply content/community/news/science-updates/2022-08-29-setiadi-blog-post.md]: Looks great!

**Comment by jeff231li** [comment]
Hmm, I'm probably missing something. Now I don't see the images.

**Comment by j-wags** [comment]
Oh, I think you may just be looking at the mark down as rendered by GitHub. That is not how the website actually renders. I can vouch that at the actual rendered website is correct, but if you want to see, follow the instructions in the repo readme and build the website using Hugo.

**Comment by jeff231li** [comment]
Ah, right. Thank you, @j-wags, for the clarification.

**Comment by diegoonolasco** [comment]
It's merged. Thanks to all of you guys!

**Comment by diegoonolasco** [comment]
I'll make a LinkedIn post about this latter today.

**Pull request merged**